packages/node_modules/@ciscospark/widget-space/README.md

```diff
@@ -131,6 +131,14 @@ If you're using our CDN, skip to the next section.
 - Add a `<script />` tag to your page to include the `bundle.js`
 - Add a `<link />` tag to include `main.css`
 
+#### Browser
+
+If you're loading an HTML file directly in the browser, use Firefox to avoid a [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) error. If you prefer Chrome, you can start it with the *--allow-file-access-from-files* flag:  
+
+```bash
+open -a "Google Chrome" --args --allow-file-access-from-files
+```
+
```

Comment:
Currently investigating this issue. This may actually be due to a change/bug in WDM.